### ISSUE TITLE
Github Actions don't run on all PRs

### ISSUE THREAD

**Issue by benibr** [ucphhpc/docker-migrid#46]
For some reason, there are PRs like #43 which did not trigger the CI test.

From https://github.com/ucphhpc/docker-migrid/blob/master/.github/workflows/ci.yml it looks like they should run on all opened PRs to me.

We should in general ensure that all merged commits have successfully completed those basic tests.

**Comment by jonasbardino** [comment]
Yeah, I can't see why it didn't and on top fought quite a bit with the CI Action these last few days. After I thought I'd finally fixed the docker ro-mount issue there the CI errors started popping up again without no obvious reason, and now just re-running the latest such failing one succeeded with no changes. I don't get it :thinking: 
I'm leaving it at that for now but perhaps someone with more insight into Githib Actions and the CI job specifically can take a look.
